src/sections/Home/Playground-home/index.js

```diff
@@ -65,18 +65,32 @@ const ViewsSectionWrapper = styled.div`
       text-align: center;
       flex-direction: column-reverse;
     }
+      
+    @media only screen and (max-width: 450px) {
+      height: 600px;
+    }
+
+    @media only screen and (max-width: 296px) {
+      height: 800px;
+    }
```

Comment:
```suggestion

```

Reply:
Sir after 300px too, this error is coming in which the text is not visible , so for that , we need to add the media query for 296px 
![Uploading Screenshot 2024-11-11 001336.png…]()

Reply:
@NishantSinghhhhh take a look at the latest deploy preview: https://6731002ae3f3370614e4a348--layer5.netlify.app/